CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_servicenow v0.4.0
+[PR #17](https://github.com/fivetran/dbt_servicenow/pull/17) includes the following updates:
+
+## Breaking Changes
+- Enabled `servicenow__user_aggregated` and `servicenow__user_enhanced` by default by changing the default `servicenow__using_roles` value to true.
+- Staging models now also enabled by default include:
+  - `stg_servicenow__sys_user_grmember`
+  - `stg_servicenow__sys_user_has_role`
+  - `stg_servicenow__sys_user_role`
+- Because this change will introduce new end model tables to users because they were initially disabled by default and changes the default behavior of a variable, we've classified this as a breaking change. 
+- Quickstart users will be able to handle disabling of these tables by [utilizing the Quickstart data model tab](https://fivetran.com/docs/using-fivetran/fivetran-dashboard/transformations#datamodels) within your ServiceNow connection, and choose to un-sync the tables required for these models.
+- Non-Quickstart users who do not have these source tables will need to re-define the variable in their `dbt_project.yml`. [See the README](https://github.com/fivetran/dbt_servicenow/blob/main/README.md#optional-step-4-additional-configurations) for more instructions. 
```

Comment:
```suggestion
- Non-Quickstart users who do not have these source tables will need to define the variable in their `dbt_project.yml`. [See the README](https://github.com/fivetran/dbt_servicenow/blob/main/README.md#optional-step-4-additional-configurations) for more instructions. 
```
It's more likely this variable wasn't defined at all in the past. So we should clarify that this needs to be defined as opposed to re-define.

Reply:
Updated